src/dss/AbaciAbstract.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// AbaciAbstract.sol -- Abaci Interfaces
+
+// Copyright (C) 2021 Maker Ecosystem Growth Holdings, INC.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.6.12;
+
+interface Abacus {
```

Comment:
I'd simplify the inheritance copying and pasting the `price` function.
Otherwise we could do the same for `wards`, `rely`, `deny` and `file`.
Not sure if we need to be that strict against the real contract.

Reply:
I was going to suggest changing the name to `AbacusAbstract` but I like just replicating the functions instead.